WebARonARKit/WebARonARKit/Shaders.metal

```diff
@@ -27,152 +28,167 @@ typedef struct {
-{
-
-    constexpr sampler colorSampler(mip_filter::linear,
-                                   mag_filter::linear,
+fragment float4 capturedImageFragmentShader(
+                                            ImageColorInOut in[[stage_in]],
+                                            texture2d<float, access::sample>
+                                            capturedImageTextureY[[texture(kTextureIndexY)]],
+                                            texture2d<float, access::sample>
+                                            capturedImageTextureCbCr[[texture(kTextureIndexCbCr)]]) {
+    constexpr sampler colorSampler(mip_filter::linear, mag_filter::linear,
                                    min_filter::linear);
-
-    const float4x4 ycbcrToRGBTransform = float4x4(
-        float4(+1.0000f, +1.0000f, +1.0000f, +0.0000f),
-        float4(+0.0000f, -0.3441f, +1.7720f, +0.0000f),
-        float4(+1.4020f, -0.7141f, +0.0000f, +0.0000f),
-        float4(-0.7010f, +0.5291f, -0.8860f, +1.0000f));
-
-    // Sample Y and CbCr textures to get the YCbCr color at the given texture coordinate
-    float4 ycbcr = float4(capturedImageTextureY.sample(colorSampler, in.texCoord).r,
+    
+    const float4x4 ycbcrToRGBTransform =
+    float4x4(float4(+1.0000f, +1.0000f, +1.0000f, +0.0000f),
+             float4(+0.0000f, -0.3441f, +1.7720f, +0.0000f),
+             float4(+1.4020f, -0.7141f, +0.0000f, +0.0000f),
+             float4(-0.7010f, +0.5291f, -0.8860f, +1.0000f));
+    
+    // Sample Y and CbCr textures to get the YCbCr color at the given texture
+    // coordinate
+    float4 ycbcr = float4(
+                          capturedImageTextureY.sample(colorSampler, in.texCoord).r,
                           capturedImageTextureCbCr.sample(colorSampler, in.texCoord).rg, 1.0);
-
+    
     // Return converted RGB color
     return ycbcrToRGBTransform * ycbcr;
 }
 
-
 typedef struct {
     float3 position[[attribute(kVertexAttributePosition)]];
     float2 texCoord[[attribute(kVertexAttributeTexcoord)]];
     half3 normal[[attribute(kVertexAttributeNormal)]];
 } Vertex;
 
-
 typedef struct {
     float4 position[[position]];
     float4 color;
     half3 eyePosition;
     half3 normal;
 } ColorInOut;
 
-
 // Anchor geometry vertex function
-vertex ColorInOut anchorGeometryVertexTransform(Vertex in[[stage_in]],
-                                                constant SharedUniforms &sharedUniforms[[buffer(kBufferIndexSharedUniforms)]],
-                                                constant InstanceUniforms *instanceUniforms[[buffer(kBufferIndexInstanceUniforms)]],
-                                                ushort vid[[vertex_id]],
-                                                ushort iid[[instance_id]])
-{
+vertex ColorInOut anchorGeometryVertexTransform(
+                                                Vertex in[[stage_in]],
```

Comment:
same thing here - move all this to the continuation indent location (4 spaces from the left edge)